electrode_models/conversion_electrode.py

```diff
@@ -1,114 +1,188 @@
 """
-    conversion_electrode.py
+    sulfur_electrode.py
 
-    Class file for conversion electrode methods. An example is lithium-sulfur, where solid S8 phase is converted to Li2S during discharge.
-
-    THe model assumes some intert, electrically conductive host phase (e.g. carbon), plus an arbitrary number of "conversion" phases.  Each conversion phase can possibly participate in reactions at the host/electrolyte interface, plus at their own electrolyte interface.
-
-    Three phase boundaries are not currently implemented, but certainly could be, at some later date.
+    Class file for air electrode methods
 """
 
 import cantera as ct
-from math import tanh
+from math import tanh, pi, isnan, exp
 import numpy as np
+import matplotlib
 
-class electrode(): 
+class electrode():
     """
-    Create an electrode object representing the single particle electrode.
+    Create an electrode object representing the air electrode.
     """
 
-    def __init__(self, input_file, inputs, sep_inputs, counter_inputs,    
+    def __init__(self, input_file, inputs, sep_inputs, an_inputs,
```

Comment:
I don't know if/how `an_inputs` gets used, but again - we need to generalize this.  I previously had `counter_inputs`, which is _a little_ better ("counter" = "other," I guess), but still not great.  Any thoughts?

Reply:
It doesn't currently get used in `conversion_electrode.py`, but it could depending on the capacity calculation (capacity due to material/species in electrolyte). I was going to ask about that in our next meeting. I see the issue of calling it explicitly `an_inputs` though. I think if we change it back to using the 'counter' terminology, possibly add `counter_ed_inputs` so that it's a little more clear?

Reply:
Sounds good to me - thanks!